docs/manual/en/introduction/Installation.html

```diff
@@ -64,7 +64,7 @@ <h2>Install from CDN or static hosting</h2>
 
 		<p>
 			The three.js library can be used without any build system, either by uploading files to your own web server or by using an existing CDN. Because the library relies on ES modules, any script that references it must use <em>type="module"</em> as shown below.
-			It is also required to define an Import Map which resolves the bare module specifier `three`.
+			It is also required to define an import map which resolves the bare module specifier `three` (and `three/addons` for the addons, see below).
```

Comment:
Please undo this change. `three/addons` can be mentioned further down the article.

Reply:
Done.